librz/core/cmd/cmd_zign.c

```diff
@@ -484,6 +484,16 @@ static int cmdOpen(void *data, const char *input) {
 	return true;
 }
 
+static RzCmdStatus print_zign_spaces(RzCore *core, RzSpaces *zs, RzOutputMode mode) {
+	RzCmdStateOutput state;
+	if (!rz_cmd_state_output_init(&state, mode)) {
+		return RZ_CMD_STATUS_INVALID;
+	}
+	rz_core_spaces_list(core, zs, &state);
+	rz_cmd_state_output_fini(&state);
+	return RZ_CMD_STATUS_OK;
+}
+
 static int cmdSpace(void *data, const char *input) {
```

Comment:
probably the all old `z` handlers can be deleted... They are converted to rzshell.

Reply:
Maybe, but it can be done separately, IMHO.